package.json

```diff
@@ -66,12 +66,13 @@
     "lint:fix": "lerna run lint:fix",
     "publish": "node scripts/publish-circleci.js",
     "test": "lerna exec --concurrency 1 --stream --bail=false -- npm run test --if-present",
-    "test:unit": "lerna exec --concurrency 1 --stream --bail=false -- npm run test:unit --if-present",
+    "test:unit": "lerna run test:unit",
```

Comment:
nx didn't play great using this `lerna exec` command but does fine with the standard `lerna run`